src/Controls/samples/Controls.Sample/MyApp.cs

```diff
@@ -1,21 +1,26 @@
 using System;
-using Maui.Controls.Sample.Services;
+using MauiSampleApp.Services;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Maui;
 using Microsoft.Maui.Controls.Compatibility;
 
-namespace Maui.Controls.Sample
+namespace MauiSampleApp
 {
-	public class MyApp : MauiApp
+	public class MyApp : IApplication
```

Comment:
This is now the "pure" MAUI app that requires nothing from controls. It works, but we don't get the XAML and the resources and things.